src/Utils/Utils.cs

```diff
@@ -34,37 +38,43 @@ public static class Utils {
             return $"https://teamtailor-production.s3.eu-west-1.amazonaws.com/image_uploads/{uuid}/original.jpg";
         }
 
-        public static string TemplateTeamTailorBody(SalesForceJob job) {
+        public static string TemplateTeamTailorBody(SalesForceJob job)
+        {
             const string templatePath = "src/templates/teamtailor-body.scriban-html";
             var template = Template.Parse(File.ReadAllText(templatePath));
             var so = new ScriptObject();
             // Parse the last answer date part, which is sent like:
-            // 2 January 2023
+            // 2000-01-01
             // We'd rather do the parsing here instead of doing formatting in SF
             var lastAnswerDatePart = job.LastAnswerDatePart;
-            DateTime lastAnswerDatePartDate = DateTime.Now;
-            DateTime.TryParseExact( // TODO: this is broken
-                    lastAnswerDatePart, 
-                    "d MMMM yyyy", 
-                    new CultureInfo("en-US"),
-                    DateTimeStyles.None,
-                    out lastAnswerDatePartDate);
+
+            var tryParseExact = DateTime.TryParseExact( // TODO: this is broken
+                lastAnswerDatePart,
+                "yyyy-MM-dd",
+                new CultureInfo("en-US"),
```

Comment:
Have no idea what and how things are used.
But generally I don't like `en-US` as they love to us `AM/PM` and love using middle-endian stirng representations in general. :smile:

Reply:
I am not really sure what `"en-US"` does here since I have already told DateTime what the format looks like :roll_eyes: but I don't think it matters what `CultureInfo` I do set here.

Reply:
Since I am no C# pro, I will take a gander at it and see what it does.

Reading the docs it is saying that `lastAnswerDatePart` has the "culture-specific formatting information" about it.

So yeah, I think and hope that it doesn't matter.